app/Models/User.php

```diff
@@ -6,7 +6,7 @@
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 
-class User extends Authenticatable
+final class User extends Authenticatable
```

Comment:
As far as I know, we usually shy away from using final for any classes unless we _really_ need to for some reason.

```suggestion
class User extends Authenticatable
```